Remove hardcoded optimization flags; add needed include and lib flags.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -1,5 +1,5 @@
-CFLAGS = -g -O2 -Wall $(shell pkg-config --cflags zlib gl freetype2)
-LDFLAGS = -lm -lglut $(shell pkg-config --libs zlib gl freetype2)
+CCFLAGS += -Wall $(shell pkg-config --cflags zlib gl freetype2) -I${LOCALBASE}/include
+LDFLAGS += -lm -lutil -L${LOCALBASE}/lib -lglut $(shell pkg-config --libs zlib gl freetype2)
 
 LIBMAN_OBJS	 = mandoc/man.o \
 			   mandoc/man_macro.o \
@@ -69,7 +69,7 @@ MANGL_SOURCES = mandoc/tree.c \
 				main.c
 
 mangl: $(COMPAT_OBJS) $(LIBMANDOC_OBJS) $(MANGL_SOURCES)
-	$(CC) $(CFLAGS) -o $@ $(COMPAT_OBJS) $(LIBMANDOC_OBJS) $(MANGL_SOURCES) $(LDFLAGS)
+	$(CC) $(CFLAGS) $(CCFLAGS) -o $@ $(COMPAT_OBJS) $(LIBMANDOC_OBJS) $(MANGL_SOURCES) $(LDFLAGS)
 
 .PHONY: install
 install: mangl
